board.php

```diff
@@ -45,12 +45,13 @@
 		// If viewing an archive page make that the title, otherwise us the name of the game
 		libHTML::starthtml(isset($_REQUEST['viewArchive'])?$_REQUEST['viewArchive']:$Game->titleBarName());
 
-		if ( isset($_REQUEST['join']) )
+		if ( isset($_REQUEST['join']) || isset($_REQUEST['joinBeta']))
 		{
 			// They will be stopped here if they're not allowed.
 			$Game->Members->join(
 				( isset($_REQUEST['gamepass']) ? $_REQUEST['gamepass'] : null ),
-				( isset($_REQUEST['countryID']) ? $_REQUEST['countryID'] : null ) );
+				( isset($_REQUEST['countryID']) ? $_REQUEST['countryID'] : null ),

```

Comment:
Since we're in PHP 7 now we can use the coalesce operator to turn `isset($_REQUEST['countryID']) ? isset($_REQUEST['countryID']) : null` into `$_REQUEST['countryID'] ?? null` which is much cleaner and easier to read.